visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -714,9 +737,6 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
             editingData[key] = newValue;
         } else {
             const asNumber = +newValue;
-            if (!Number.isFinite(asNumber)) {
```

Comment:
why was the NaN check removed?

Reply:
it was just debugging raising an alert. We can add a warning or alert.

Reply:
a warning might be nice